### PR TITLE
Add to calendar

### LINKED ISSUE #97: Calendar to iCal / Google Calendar

### PR DESCRIPTION
**Pull request by cmw0513**
closes #97

### CODE REVIEW

**Comment by cmw0513** [comment]
This is an option that is free for personal use. It's $5/month for commercial use or we can find another option if we want to go that route

**Comment by lienvdsteen** [comment]
Commercial use is as soon as we ask money for our product too?

File: app/views/games/show.html.erb [hunk 1]
```diff
@@ -29,6 +29,17 @@
     <div class="col-sm-4">
       <!-- TODO: if logged in player, check if they already confirmed or not  -->
     </div>
+    <div title="Add to Calendar" class="addeventatc">
+      Add to Calendar
+      <span class="start"><%= @game.date_time.strftime("%d/%m/%Y %H:%M") %></span>
```
Comment (lienvdsteen) [inline app/views/games/show.html.erb]: we probably should add a helper method that decides how to format dates

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,6 +6,8 @@
     <%= csrf_meta_tags %>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <!-- AddEvent -->
+    <script type="text/javascript" src="https://addevent.com/libs/atc/1.6.1/atc.min.js"></script>
```
Comment (lienvdsteen) [inline app/views/layouts/application.html.erb]: it's better to add script tags add the bottom of the file for performance reasons before `<%= yield(:after_js) %>`

**Comment by cmw0513** [comment]
Yes, once we start charging money for it we are using it commercially. Fixed the two things you mentioned, thanks!

**Comment by lienvdsteen** [comment]
ok then we should keep that in mind if we charge for the app! otherwise you can merge

**Pull request merged**